ownCloud/Server List/ServerListTableViewController.swift

```diff
@@ -345,9 +340,77 @@ class ServerListTableViewController: UITableViewController, Themeable {
 		}
 	}
 
+	func setupAddBookmarkActions() {
+		if VendorServices.shared.canAddAccount {
+			var addServerBarButtonItem: UIBarButtonItem!
+
+			if #available(iOS 14.0, *), OCBookmarkManager.shared.bookmarks.count == 0 {
+				addServerBarButtonItem = UIBarButtonItem(systemItem: .add, primaryAction: nil, menu: createAddBookmarkMenu())
+				welcomeAddServerButton.menu = createAddBookmarkMenu()
```

Comment:
@hosy is menu support not available on iOS13?

Reply:
Not for `UIBarButtonItem` and `UIButton` only for contextual menus